Updated OCM-role requirements throughout the ROSA docs (#107845)
openshift/openshift-docs: modules/rosa-hcp-creating-a-cluster-using-defaults-ocm.adoc

====
. On the *Machine pools* page, select your created VPC and at least one of your private subnet IDs.

. On the *Network configuration* page, if your cluster is publicly available, provide your public subnet ID.

. To deploy a cluster quickly, leave the default options in the *Cluster settings*, *Networking*, *Cluster roles and policies*, and *Cluster updates* pages and click *Next* on each page.

. On the *Review your {product-title} cluster* page, review the summary of your selections and click *Create cluster* to start the installation.
+
. Optional: On the *Overview* tab, you can enable the delete protection feature by selecting *Enable*, which is located directly under *Delete Protection: Disabled*. This prevents your cluster from being deleted. To disable delete protection, select *Disable*.
By default, clusters are created with the delete protection feature disabled.
+

.Verification

> * You can check the progress of the installation in the *Overview* page for your cluster. You can view the installation logs on the same page. Your cluster is ready when the *Status* in the *Details* section of the page is listed as *Ready*.

Reviewer: Maybe make these steps? 

```suggestion
. Go to the *Overview* page for your cluster. 
.  Check the progress of the installation and view the installation logs.  
. Go to  *Details* > *Status* and confirm that your cluster is listed as *Ready*. 
```

Reviewer: Also, bold and monospacing confuses me and is a pain in the neck, lol. But since we're not clicking or selecting "Ready" perhaps monospace is more appropriate than bold, since we use bold to prompt users to select that interface, 'Ready'. Kind of a matter of opinion though.
